src/sonic-bgpcfgd/bgpcfgd/managers_bgp.py

```diff
@@ -304,10 +304,11 @@ def apply_op(self, cmd, vrf):
         :return: True if no errors, False if there are errors
         """
         bgp_asn = self.directory.get_slot("CONFIG_DB", swsscommon.CFG_DEVICE_METADATA_TABLE_NAME)["localhost"]["bgp_asn"]
+        enable_bgp_suppress_fib_pending_cmd = 'bgp suppress-fib-pending'
         if vrf == 'default':
-            cmd = ('router bgp %s\n' % bgp_asn) + cmd
+            cmd = ('router bgp %s\n %s\n' % (bgp_asn, enable_bgp_suppress_fib_pending_cmd)) + cmd
```

Comment:
I'm not sure if I understand this correctly, this is a feature enablement or this is a section? Why need to append the 'bgp suppress-fib-pending' for each time apply op?

Reply:
@StormLiangMS In current design of this daemon, the bgp router is configured with "router bgp %s vrf" command on first appearence of Vrf and enters the context of that router. Then, "router bgp %s vrf" is just used to enter the context of vrf router. There are no global BGP configs applied for bgp router in vrf in this daemon. I decided to put here enablement of suppression as the easiest solution I could think of. Another way would be create a separate manager that will listen for vrf table updates and apply this config once per VRF in case there are BGP_NEIGHBOR entries that reference this VRF. Let me know your opinion.